python/pip.bzl

```diff
@@ -34,22 +35,35 @@ def _pip_import_impl(repository_ctx):
   if result.return_code:
     fail("pip_import failed: %s (%s)" % (result.stdout, result.stderr))
 
+_shared_attrs = {
+    "requirements": attr.label(
+        allow_files = True,
+        mandatory = True,
+        single_file = True,
+    ),
+    "_script": attr.label(
+        executable = True,
+        default = Label("//tools:piptool.par"),
+        cfg = "host",
+    ),
+}
+
+def _pip_import_impl(repository_ctx):
+    _shared_pip_import_impl("python2", repository_ctx)
```

Comment:
The default `pip_import()` should just use the system python. ie. `s/python2/python/`. A `pip2_import()` could be added if needed that did this, I suppose.